lib/load-plugins.js

```diff
@@ -1,13 +1,51 @@
-// Load all plugins from /lib/plugins/*/index.js.
-// For more information about dynamic require see
-// https://webpack.github.io/docs/context.html
+import BowerManifest from './plugins/bower-manifest.js';
+import Composer from './plugins/composer-manifest.js';
+import CSS from './plugins/css.js';
+import Docker from './plugins/docker.js';
+import DotNetCore from './plugins/dot-net-core.js';
+import DotNet from './plugins/dot-net.js';
+import Rubygems from './plugins/gemfile-manifest.js';
+import Go from './plugins/go.js';
+import Haskell from './plugins/haskell.js';
+import Homebrew from './plugins/homebrew-manifest.js';
+import HTML from './plugins/html.js';
+import Java from './plugins/java.js';
+import JavaScript from './plugins/javascript.js';
+import Less from './plugins/less.js';
+import NodejsRelativePath from './plugins/nodejs-relative-path.js';
+import NpmManifest from './plugins/npm-manifest.js';
+import Python from './plugins/python.js';
+import RequirementsTxt from './plugins/requirements-txt.js';
+import Ruby from './plugins/ruby.js';
+import Rust from './plugins/rust.js';
+import Sass from './plugins/sass.js';
+import TypeScript from './plugins/typescript.js';
+import Vim from './plugins/vim.js';
 
 export default function loadPlugins() {
-  function requireAll(requireContext) {
-    return requireContext
-      .keys()
-      .map(pluginPath => requireContext(pluginPath).default);
-  }
-
-  return requireAll(require.context('./plugins', false, /[^/]*.js$/));
+  return [
+    BowerManifest,
+    Composer,
+    CSS,
+    Docker,
+    DotNetCore,
+    DotNet,
+    Rubygems,
+    Go,
+    Haskell,
+    Homebrew,
+    HTML,
+    Java,
+    JavaScript,
+    Less,
+    NodejsRelativePath,
+    NpmManifest,
+    Python,
+    RequirementsTxt,
+    Ruby,
+    Rust,
+    Sass,
+    TypeScript,
+    Vim,
+  ];
```

Comment:
Instead of having to repeat the name of every `import`, we could use `export { default as BowerManifest } from './plugins/bower-manifest.js';`/etc instead of the `import`s at the top. Other modules would then have to use `import * as loadPlugins from '../lib/load-plugins.js';` and `Object.values(loadPlugins)` instead of `loadPlugins()`. I pushed an example change to https://github.com/josephfrazier/browser-extension/commit/d471abac74463b5fb8b5f35f0567e6aa6f4a4fb2, but I'm not sure if the added overhead of `Object.values` is worth not having to repeat the plugin names.

Reply:
Yeah that make sense. Even with the small down side of using `Object.values` it's fine. Will cherry pick your commit.